app/views/other/gambling/mgd/doYouHaveMGDRegistrationNo.scala.html

```diff
@@ -0,0 +1,43 @@
+@*
```

Comment:
Rename this file from `doYouHaveMGDRegistrationNo` to `doYouHaveMGDRegistration`